site/src/Librecores/ProjectRepoBundle/Entity/OrganizationRepository.php

```diff
@@ -12,42 +12,23 @@
 class OrganizationRepository extends EntityRepository
 {
     /**
-     * Find all the organizations that a user owns.
-     *
-     * @param User $user
-     * @return ArrayCollection of Organizations
-     */
-    public function findAllByOwnerOrderedByName(User $user)
-    {
-        return $this->getEntityManager()
-            ->createQueryBuilder()
-            ->select('o')
-            ->from('LibrecoresProjectRepoBundle:Organization', 'o')
-            ->where('o.owner = :owner')
-            ->setParameter('owner', $user->getId())
-            ->orderBy('o.name', 'ASC')
-            ->getQuery()
-            ->getResult();
-    }
-
-    /**
      * Find all the organizations that a user is a member of.
      *
      * @param User $user
-     * @return ArrayCollection of Organizations
+     * @return array of Organizations
```

Comment:
Agreed, I made the change to make a code check warning pass but what you mention is the better fix which keeps the type and I'm positive it will pass the check as well.